app/controllers/collection_numbers_controller.rb

```diff
@@ -104,6 +104,15 @@ def new
   def create
     store_location
     pass_query_params
+    @layout = calc_layout_params
```

Comment:
Do we still need `@layout`?
I'm currently finding it only in ImageController; will it go away there?

Reply:
You're right - Probably not.